### PR TITLE
More reaction messages

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

More reaction messages when mixing chems, basically fluff

## Why It's Good For The Game

Makes mixing chems have more feedback, as well as helps with people being able to tell what they make

## Changelog
:cl:
spellcheck: Lots more chem reaction messages 
/:cl:

### CODE REVIEW

**Comment by chefferz** [comment]
Misspelling.

Horrible
Emits

**Comment by Ghommie** [comment]
Please don't try adding more flawed descriptions and messages to the game so bluntly if you are so well aware of your own limitations with the english language, you'll most likely just end up running into a stone wall. That's said, we are better off without this PR.

**Comment by nik707** [comment]
Go ahead and find someone to proofread it, thanks

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 1]
```diff
@@ -26,27 +26,31 @@
...
 	required_reagents = list(/datum/reagent/medicine/ephedrine = 1, /datum/reagent/iodine = 1, /datum/reagent/phosphorus = 1, /datum/reagent/hydrogen = 1)
+	mix_message = "<span class='danger'>Large crystalline shards form.</span>"
 	required_temp = 374
 
 /datum/chemical_reaction/bath_salts
 	name = "bath_salts"
 	id = /datum/reagent/drug/bath_salts
 	results = list(/datum/reagent/drug/bath_salts = 7)
 	required_reagents = list(/datum/reagent/toxin/bad_food = 1, /datum/reagent/saltpetre = 1, /datum/reagent/consumable/nutriment = 1, /datum/reagent/space_cleaner = 1, /datum/reagent/consumable/enzyme = 1, /datum/reagent/consumable/tea = 1, /datum/reagent/mercury = 1)
+	mix_message = "Hard chunks fall out of solution."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/drugs.dm]:
```suggestion
	mix_message = "Hard chunks fall out of the solution."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 2]
```diff
@@ -4,53 +4,62 @@
...
 	results = list(/datum/reagent/glycerol = 1)
 	required_reagents = list(/datum/reagent/consumable/cornoil = 3, /datum/reagent/toxin/acid = 1)
+	mix_message = "The mixture darkens as it becomes thick."
 
 /datum/chemical_reaction/sodiumchloride
 	name = "Sodium Chloride"
 	id = "sodiumchloride"
 	results = list(/datum/reagent/consumable/sodiumchloride = 3)
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/sodium = 1, /datum/reagent/chlorine = 1)
+	mix_message = "The mix drys rapidly forming chunks."
```
Comment (nik707) [inline code/modules/reagents/chemistry/recipes/others.dm]: dries

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 3]
```diff
@@ -5,30 +5,35 @@
 	results = list(/datum/reagent/medicine/leporazine = 2)
 	required_reagents = list(/datum/reagent/silicon = 1, /datum/reagent/copper = 1)
 	required_catalysts = list(/datum/reagent/toxin/plasma = 5)
+	mix_message = "The liquid becomes a light pink"
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The liquid becomes a light pink colour."
```

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 4]
```diff
@@ -5,30 +5,35 @@
...
 
 /datum/chemical_reaction/rezadone
 	name = "Rezadone"
 	id = /datum/reagent/medicine/rezadone
 	results = list(/datum/reagent/medicine/rezadone = 3)
 	required_reagents = list(/datum/reagent/toxin/carpotoxin = 1, /datum/reagent/cryptobiolin = 1, /datum/reagent/copper = 1)
+	mix_message = "The liquid starts to harden and become dry"
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The liquid starts to harden and become dry."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 5]
```diff
@@ -123,12 +136,14 @@
...
 	required_reagents = list(/datum/reagent/carbon = 1, /datum/reagent/oxygen = 2)
 	required_temp = 777 // pure carbon isn't especially reactive.
+	mix_message = "The mix burns up into forming something unknown."
 
 /datum/chemical_reaction/nitrous_oxide
 	name = "Nitrous Oxide"
 	id = /datum/reagent/nitrous_oxide
 	results = list(/datum/reagent/nitrous_oxide = 5)
 	required_reagents = list(/datum/reagent/ammonia = 2, /datum/reagent/nitrogen = 1, /datum/reagent/oxygen = 2)
+	mix_message = "The mix slowly burns gasifying."
```
Comment (nik707) [inline code/modules/reagents/chemistry/recipes/others.dm]: burns, gasifying.

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 6]
```diff
@@ -5,30 +5,35 @@
...
 
 /datum/chemical_reaction/spaceacillin
 	name = "Spaceacillin"
 	id = /datum/reagent/medicine/spaceacillin
 	results = list(/datum/reagent/medicine/spaceacillin = 2)
 	required_reagents = list(/datum/reagent/cryptobiolin = 1, /datum/reagent/medicine/epinephrine = 1)
+	mix_message = "The liquid mixes smoothly"
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The liquid mixes smoothly."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 7]
```diff
@@ -445,42 +466,49 @@
...
 	results = list(/datum/reagent/diethylamine = 2)
 	required_reagents = list (/datum/reagent/ammonia = 1, /datum/reagent/consumable/ethanol = 1)
+	mix_message = "The liquid starts to emit a bad odor."
 
 /datum/chemical_reaction/space_cleaner
 	name = "Space cleaner"
 	id = /datum/reagent/space_cleaner
 	results = list(/datum/reagent/space_cleaner = 2)
 	required_reagents = list(/datum/reagent/ammonia = 1, /datum/reagent/water = 1)
+	mix_message = "The liquid starts to emit lemony smell." //Joke on lemony sented cleaners
```
Comment (nik707) [inline code/modules/reagents/chemistry/recipes/others.dm]: a lemony smell

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 8]
```diff
@@ -190,18 +206,21 @@
 	id = /datum/reagent/medicine/atropine
 	results = list(/datum/reagent/medicine/atropine = 5)
 	required_reagents = list(/datum/reagent/consumable/ethanol = 1, /datum/reagent/acetone = 1, /datum/reagent/diethylamine = 1, /datum/reagent/phenol = 1, /datum/reagent/toxin/acid = 1)
+	mix_message = "The liquid starts to blacken quickly."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The liquid quickly blackens."
```

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 9]
```diff
@@ -190,18 +206,21 @@
...
 	results = list(/datum/reagent/medicine/epinephrine = 6)
 	required_reagents = list(/datum/reagent/phenol = 1, /datum/reagent/acetone = 1, /datum/reagent/diethylamine = 1, /datum/reagent/oxygen = 1, /datum/reagent/chlorine = 1, /datum/reagent/hydrogen = 1)
+	mix_message = "The mixture sputters loudly before calming down."
 
 /datum/chemical_reaction/strange_reagent
 	name = "Strange Reagent"
 	id = /datum/reagent/medicine/strange_reagent
 	results = list(/datum/reagent/medicine/strange_reagent = 3)
 	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/water/holywater = 1, /datum/reagent/toxin/mutagen = 1)
+	mix_message = "The mixture sputters loudly and smells sickly."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The mixture sputters loudly and emits a sickly smell."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 10]
```diff
@@ -610,27 +652,31 @@
 	id = /datum/reagent/pax
 	results = list(/datum/reagent/pax = 3)
 	required_reagents  = list(/datum/reagent/toxin/mindbreaker = 1, /datum/reagent/medicine/synaptizine = 1, /datum/reagent/water = 1)
+	mix_message = "The liquid starts to look peacefull."
```
Comment (nik707) [inline code/modules/reagents/chemistry/recipes/others.dm]: peaceful

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 11]
```diff
@@ -215,91 +234,106 @@
...
 	results = list(/datum/reagent/medicine/corazone = 3)
 	required_reagents = list(/datum/reagent/phenol = 2, /datum/reagent/lithium = 1)
+	mix_message = "The mixture becomes a light red colour."
 
 /datum/chemical_reaction/morphine
 	name = "Morphine"
 	id = /datum/reagent/medicine/morphine
 	results = list(/datum/reagent/medicine/morphine = 2)
 	required_reagents = list(/datum/reagent/carbon = 2, /datum/reagent/hydrogen = 2, /datum/reagent/consumable/ethanol = 1, /datum/reagent/oxygen = 1)
+	mix_message = "The liquid densifies as its heated."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The liquid densifies as it's heated."
```

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 12]
```diff
@@ -308,9 +342,11 @@
 	results = list(/datum/reagent/medicine/modafinil = 5)
 	required_reagents = list(/datum/reagent/diethylamine = 1, /datum/reagent/ammonia = 1, /datum/reagent/phenol = 1, /datum/reagent/acetone = 1, /datum/reagent/toxin/acid = 1)
 	required_catalysts = list(/datum/reagent/bromine = 1) // as close to the real world synthesis as possible
+	mix_message = "The liquid densifies as its heated."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	mix_message = "The liquid densifies as it's heated."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 13]
```diff
@@ -4,53 +4,62 @@
...
 	id = "sodiumchloride"
 	results = list(/datum/reagent/consumable/sodiumchloride = 3)
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/sodium = 1, /datum/reagent/chlorine = 1)
+	mix_message = "The mix drys rapidly forming chunks."
 
 /datum/chemical_reaction/plasmasolidification
 	name = "Solid Plasma"
 	id = "solidplasma"
 	required_reagents = list(/datum/reagent/iron = 5, /datum/reagent/consumable/frostoil = 5, /datum/reagent/toxin/plasma = 20)
+	mix_message = "The mix drys rapidly forming large crystals of plasma."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix dries, rapidly forming large crystals of plasma."
```

File: code/modules/reagents/chemistry/recipes/others.dm (hunk 2 above)
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix dries, rapidly forming chunks."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 14]
```diff
@@ -62,6 +71,7 @@
 	name = "Solid Gold"
 	id = "solidgold"
 	required_reagents = list(/datum/reagent/consumable/frostoil = 5, /datum/reagent/gold = 20, /datum/reagent/iron = 1)
+	mix_message = "The mix drys rapidly forming large crystals of gold."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix dries, rapidly forming large crystals of gold."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 15]
```diff
@@ -80,6 +90,7 @@
 	id = "soapification"
 	required_reagents = list(/datum/reagent/liquidgibs = 10, /datum/reagent/lye  = 10) // requires two scooped gib tiles
 	required_temp = 374
+	mix_message = "The mix drys rapidly forming large chunks of soap."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix dries, rapidly forming large chunks of soap."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 16]
```diff
@@ -98,6 +109,7 @@
 	id = "candlefication"
 	required_reagents = list(/datum/reagent/liquidgibs = 5, /datum/reagent/oxygen  = 5)
 	required_temp = 374
+	mix_message = "The mix drys rapidly forming large chunks of wax."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix dries, rapidly forming large chunks of wax."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 17]
```diff
@@ -109,6 +121,7 @@
 	name = "Meatification"
 	id = "meatification"
 	required_reagents = list(/datum/reagent/liquidgibs = 10, /datum/reagent/consumable/nutriment = 10, /datum/reagent/carbon = 10)
+	mix_message = "The mix thickens rapidly forming large chunks of meat."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix thickens, rapidly forming large chunks of meat."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 18]
```diff
@@ -123,12 +136,14 @@
 	results = list(/datum/reagent/carbondioxide = 3)
 	required_reagents = list(/datum/reagent/carbon = 1, /datum/reagent/oxygen = 2)
 	required_temp = 777 // pure carbon isn't especially reactive.
+	mix_message = "The mix burns up into forming something unknown."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix burns up, forming into something unknown."
```

File: code/modules/reagents/chemistry/recipes/others.dm (hunk 5 above)
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix slowly burns, turning into a gas."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 19]
```diff
@@ -489,31 +517,36 @@
...
 	results = list(/datum/reagent/phenol = 3)
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/chlorine = 1, /datum/reagent/oil = 1)
+	mix_message = "The mix becomes runny."
 
 /datum/chemical_reaction/ash
 	name = "Ash"
 	id = /datum/reagent/ash
 	results = list(/datum/reagent/ash = 1)
 	required_reagents = list(/datum/reagent/oil = 1)
+	mix_message = "The liquid burns up making a dust."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The liquid burns up into dust."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 20]
```diff
@@ -489,31 +517,36 @@
...
 	required_temp = 480
 
 /datum/chemical_reaction/colorful_reagent
 	name = "colorful_reagent"
 	id = /datum/reagent/colorful_reagent
 	results = list(/datum/reagent/colorful_reagent = 5)
 	required_reagents = list(/datum/reagent/stable_plasma = 1, /datum/reagent/radium = 1, /datum/reagent/drug/space_drugs = 1, /datum/reagent/medicine/cryoxadone = 1, /datum/reagent/consumable/triple_citrus = 1)
+	mix_message = "The mix colourful staining the container."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix becomes colourful, staining the container."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 21]
```diff
@@ -551,54 +584,63 @@
 	id = /datum/reagent/hair_dye
 	results = list(/datum/reagent/hair_dye = 5)
 	required_reagents = list(/datum/reagent/colorful_reagent = 1, /datum/reagent/radium = 1, /datum/reagent/drug/space_drugs = 1)
+	mix_message = "The mix starts to leave long strands along the container walls."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix starts to leave long strands along the container's walls."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 22]
```diff
@@ -551,54 +584,63 @@
...
 
 /datum/chemical_reaction/barbers_aid
 	name = "barbers_aid"
 	id = /datum/reagent/barbers_aid
 	results = list(/datum/reagent/barbers_aid = 5)
 	required_reagents = list(/datum/reagent/carpet = 1, /datum/reagent/radium = 1, /datum/reagent/drug/space_drugs = 1)
+	mix_message = "The mix starts to leave long strands along the container walls."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix starts to leave long strands along the container's walls."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 23]
```diff
@@ -551,54 +584,63 @@
...
 
 /datum/chemical_reaction/concentrated_barbers_aid
 	name = "concentrated_barbers_aid"
 	id = /datum/reagent/concentrated_barbers_aid
 	results = list(/datum/reagent/concentrated_barbers_aid = 2)
 	required_reagents = list(/datum/reagent/barbers_aid = 1, /datum/reagent/toxin/mutagen = 1)
+	mix_message = "The mix starts to leave long strands along the container walls."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The mix starts to leave long strands along the container's walls."
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 24]
```diff
@@ -551,54 +584,63 @@
...
 	results = list(/datum/reagent/lye = 2)
 	required_reagents = list(/datum/reagent/ash = 1, /datum/reagent/water = 1, /datum/reagent/carbon = 1)
+	mix_message = "The liquid starts to become runny and turn chalky white."
 
 /datum/chemical_reaction/royal_bee_jelly
 	name = "royal bee jelly"
 	id = /datum/reagent/royal_bee_jelly
 	results = list(/datum/reagent/royal_bee_jelly = 5)
 	required_reagents = list(/datum/reagent/toxin/mutagen = 10, /datum/reagent/consumable/honey = 40)
+	mix_message = "The liquid starts to stick to the container walls."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	mix_message = "The liquid starts to stick to the container's walls."
```

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 25]
```diff
@@ -4,40 +4,46 @@
...
 	results = list(/datum/reagent/toxin/itching_powder = 3)
 	required_reagents = list(/datum/reagent/fuel = 1, /datum/reagent/ammonia = 1, /datum/reagent/medicine/charcoal = 1)
+	mix_message = "<span class='danger'>A dusty powder forms.</span>"
 
 /datum/chemical_reaction/facid
 	name = "Fluorosulfuric acid"
 	id = /datum/reagent/toxin/acid/fluacid
 	results = list(/datum/reagent/toxin/acid/fluacid = 4)
 	required_reagents = list(/datum/reagent/toxin/acid = 1, /datum/reagent/fluorine = 1, /datum/reagent/hydrogen = 1, /datum/reagent/potassium = 1)
+	mix_message = "<span class='danger'>A colourless liquid froms.</span>"
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/toxins.dm]:
```suggestion
	mix_message = "<span class='danger'>A colourless liquid forms.</span>"
```

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 26]
```diff
@@ -50,49 +56,56 @@
...
 	required_reagents = list(/datum/reagent/silicon = 1, /datum/reagent/hydrogen = 1, /datum/reagent/medicine/charcoal = 1)
+	mix_message = "The liquid fluffs up quickly."
 
 /datum/chemical_reaction/heparin
 	name = "Heparin"
 	id = /datum/reagent/toxin/heparin
 	results = list(/datum/reagent/toxin/heparin = 4)
 	required_reagents = list(/datum/reagent/toxin/formaldehyde = 1, /datum/reagent/sodium = 1, /datum/reagent/chlorine = 1, /datum/reagent/lithium = 1)
-	mix_message = "<span class='danger'>The mixture thins and loses all color.</span>"
+	mix_message = "The mixture thins and loses all color."
```
Comment (Linzolle) [inline code/modules/reagents/chemistry/recipes/toxins.dm]: What's the reason for removing the danger class?

Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: the reaction has no danger to it as theirs no heparin bomb act / not that bad of a chem...
Ill add the danger class span back in

**Pull request closed without merging**